android-json-form-wizard/src/main/java/com/vijay/jsonwizard/fragments/JsonFormFragment.java

```diff
@@ -106,6 +136,44 @@ private void setupToolbarBackButton() {
 
     }
 
+    protected void initializeBottomNavigation(JSONObject step, View rootView) {
+        bottomNavigation.setVisibility(View.VISIBLE);
+        if (step.has(PREVIOUS)) {
+            previousButton.setVisibility(View.VISIBLE);
+            if (step.has(PREVIOUS_LABEL)) {
+                previousButton.setText(step.optString(PREVIOUS_LABEL));
+            }
+        }
+
+        if (step.has(NEXT)) {
+            nextButton.setVisibility(View.VISIBLE);
+            if (step.has(NEXT_LABEL)) {
+                nextButton.setText(step.optString(NEXT_LABEL));
+            }
+        }  else if (step.optString(NEXT_TYPE).equalsIgnoreCase(SUBMIT)) {
+            nextButton.setTag(R.id.submit, true);
+            nextButton.setVisibility(View.VISIBLE);
+        }
+
+        if (step.has(BOTTOM_NAVIGATION_ORIENTATION)) {
+            // layout orientation
+            int orientation = "vertical".equals(step.optString(BOTTOM_NAVIGATION_ORIENTATION)) ? LinearLayout.VERTICAL : LinearLayout.HORIZONTAL;
+            bottomNavigation.setOrientation(orientation);
+            bottomNavigation.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT));
+            bottomNavigation.removeView(previousButton);
+            bottomNavigation.addView(previousButton);
+            // nav btn params
+            LinearLayout.LayoutParams params = new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT);
+            int layoutMargin = Utils.pixelToDp(15, getContext());
```

Comment:
Maybe you use the values defined in the dimes.xml i.e `bottom_navigation_margin`

Reply:
Done.